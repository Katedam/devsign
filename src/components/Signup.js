import React from 'react';
import { Form, Input, Button, StyledHeader, Label } from '../styles/mainStyle';

function Signup() {
  return (
    <>
    <StyledHeader>
      <h1>Chirp</h1>
    </StyledHeader>
    <Form>
      <Label htmlFor="name">Name</Label>
      <Input readOnly={true} type="text" name="name" id="name" value="name"/>

      <Label htmlFor="handle">Chirp Handle</Label>
      <Input readOnly={true} type="text" name="handle" id="handle" value="handle"/>

      <Label htmlFor="password">password</Label>
      <Input readOnly={true} type="password" name="password" id="password" value="password"/>

      <Label htmlFor="sign-in"></Label>
      <Button id="sign-in">Sign In</Button>
    </Form>
    </>
  );
}

export default Signup;
