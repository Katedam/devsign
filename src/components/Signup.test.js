import React from 'react';
import Signup from './Signup.js';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';

describe('Signup component', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <MemoryRouter>
        <Signup />
      </MemoryRouter>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
