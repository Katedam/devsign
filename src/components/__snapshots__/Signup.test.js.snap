// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signup Snapshot matches a snapshot 1`] = `
Array [
  <header
    className="sc-bwzfXH ikzJqp"
  >
    <h1>
      Chirp
    </h1>
  </header>,
  <form
    className="sc-bxivhb dBAuwS"
  >
    <label
      className="sc-ifAKCX hrWLvt"
      htmlFor="name"
    >
      Name
    </label>
    <input
      className="sc-bZQynM dqSVUO"
      id="name"
      name="name"
      readOnly={true}
      type="text"
      value="name"
    />
    <label
      className="sc-ifAKCX hrWLvt"
      htmlFor="handle"
    >
      Chirp Handle
    </label>
    <input
      className="sc-bZQynM dqSVUO"
      id="handle"
      name="handle"
      readOnly={true}
      type="text"
      value="handle"
    />
    <label
      className="sc-ifAKCX hrWLvt"
      htmlFor="password"
    >
      password
    </label>
    <input
      className="sc-bZQynM dqSVUO"
      id="password"
      name="password"
      readOnly={true}
      type="password"
      value="password"
    />
    <label
      className="sc-ifAKCX hrWLvt"
      htmlFor="sign-in"
    />
    <button
      className="sc-gzVnrw ccvAWg"
      id="sign-in"
    >
      Sign In
    </button>
  </form>,
]
`;
